microcosm_postgres/encryption/v2/column.py

```diff
@@ -0,0 +1,38 @@
+from sqlalchemy.ext.hybrid import hybrid_property
+
+from .encoders import Encoder
+from .encryptors import Encryptor
+
+
+def encryption(key: str, encryptor: Encryptor, encoder: Encoder) -> hybrid_property:
+    """
+    Switches between encrypted and plaintext values based on the client_id.
+
+    Queries on the encryption field will only be performed on the unencrypted rows.
+    """
+    encrypted_field = f"{key}_encrypted"
```

Comment:
I'm wondering - what if people mistype `_encrypted` / `_decrypted`? What error message displays?

Reply:
Think you'll get an attribute error when you access the encrypted field